portfolio/src/main/webapp/index.html

```diff
@@ -121,16 +118,19 @@ <h1>Guinea Pig Gallery</h1>
   <a class="next" onclick="plusSlides(1)">&#10095;</a>
 
   </div>
+
+  <div id="comment-container">
+  <h1>Comments</h1>
+  <p>Share your thoughts:</p>
   <body onload="getComments()">
-    <h1>Comments</h1>
     <form action="/data" method="POST">
-      <p>Share your thoughts:</p>
-      <textarea name="text-input">Looks awesome!</textarea>
+      <textarea name="comment" rows="3" cols="35">Your Name: Your Comment</textarea>
       <br/><br/>
-
       <input type="submit" />
     </form>
-  </body>
+    <hr/>
+    <ul id="comment-list"></ul>
+    </div>
```

Comment:
The indentation seems a bit off. I am not super familiar with HTML myself, but I think proper indentation is particularly important for reading:)